package.json

```diff
@@ -37,6 +37,7 @@
   "devDependencies": {
     "eslint": "4.19.1",
     "eslint-plugin-prettier": "2.6.0",
+    "@types/node": "^10.0.6",
```

Comment:
in `dependencies`?

Reply:
as i commented above, arguably there's no reason to add this dependency at all -- it provides no real benefit but adds maintenance cost (and might erroneously stick node globals into the typescript namespace in places where this might not be desired!)